tests/integration/components/cta-ember-community-survey-test.js

```diff
@@ -0,0 +1,42 @@
+import { click, render } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+import { setupRenderingTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+
+module('Integration | Component | cta-ember-community-survey', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('We can display a survey reminder', async function (assert) {
+    this.owner.setupRouter()
```

Comment:
without it, the test does not emit `href` attribute
https://github.com/emberjs/ember.js/issues/16904

Reply:
Yep, that's correct. I didn't mention in the issue, but we need to write `this.owner.setupRouter();` in a rendering test until we upgrade `ember-source` to v3.24.